tasks/build-website-testing.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require('gulp');
+var config = require('./config.js');
```

Comment:
ok, the right is:

```js
var gulp =   require('gulp');
var config = require('./config.js');
``
or 

```js
var gulp    = require('gulp');
var config = require('./config.js');
```

reading the key-spacing align value, I believe the rule get only values inside a object, not variable assigns, because of that rule is called key and not variable-spacing :D

Eslint probaly not can lint our code because we dont have a rule in .eslint, at now, to force this, I believe